### ISSUE TITLE
[UI/UX] Reading position

### ISSUE THREAD

**Issue by llemeurfr** [readium/readium-desktop#23]
To be innovative we should have some kind of an indicator to display the current reading position (how much you have "read", depending on how far you have scrolled down an article).

Read this [interesting post on the subject](https://css-tricks.com/reading-position-indicator/).
I found referece to this in another interesting post about [pagination and word processors](https://www.adamhyde.net/towards-web-based-word-processors-p3/).

This is a place for your ideas about the best way to indicate his reading position to the user.

**Comment by JayPanoz** [comment]
A few (technical) notes: 

- [Scrolling on the web: a primer](https://blogs.windows.com/msedgedev/2017/03/08/scrolling-on-the-web/) can extend the [Performance section](https://css-tricks.com/reading-position-indicator/#article-header-id-12) in the first link as it explains why scrolling is a lot more complex that it seems;
- The demo in the first link is somewhat possible using pure CSS ([see this codepen])(https://codepen.io/MadeByMike/pen/ZOrEmr) but it’s quite a dirty trick, really.

**Comment by llemeurfr** [comment]
Done in March 2019.
